### PR TITLE
 Add definition for MAX_SPI_SPEED to limit spi seed if defined for target

### PR DESCRIPTION
**Pull request by marbalon**
This is simple solution for problem described here #3012

### CODE REVIEW

File: src/main/drivers/bus_busdev_spi.c [hunk 1]
```diff
@@ -33,6 +33,12 @@ void spiBusSetSpeed(const busDevice_t * dev, busSpeed_e speed)
 {
     const SPIClockSpeed_e spiClock[] = { SPI_CLOCK_INITIALIZATON, SPI_CLOCK_SLOW, SPI_CLOCK_STANDARD, SPI_CLOCK_FAST, SPI_CLOCK_ULTRAFAST };
     SPI_TypeDef * instance = spiInstanceByDevice(dev->busdev.spi.spiBus);
+
+#ifdef MAX_SPI_SPEED
```
Comment (fiam) [inline src/main/drivers/bus_busdev_spi.c]: Please, make this `BUS_SPI_MAX_SPEED`

Comment (fiam) [reply src/main/drivers/bus_busdev_spi.c]: Sorry, BUS_SPI_SPEED_MAX

**Comment by fiam** [comment]
Perfect, thanks! I'll merge as soon as the tests pass.

**Comment by marbalon** [comment]
Thanks!

**Pull request merged**